processor/src/main/java/org/mapstruct/ap/internal/model/PropertyMapping.java

```diff
@@ -18,17 +18,11 @@
  */
 package org.mapstruct.ap.internal.model;
 
-import static org.mapstruct.ap.internal.model.assignment.Assignment.AssignmentType.DIRECT;
```

Comment:
Apart from the imports and one empty line there are no changes to this file. Can you please revert all changes to this one?